rtptransceiver.go

```diff
@@ -260,10 +260,9 @@ func satisfyTypeAndDirection(
 			return []RTPTransceiverDirection{
 				RTPTransceiverDirectionRecvonly,
 				RTPTransceiverDirectionSendrecv,
-				RTPTransceiverDirectionSendonly,
```

Comment:
```
 If an offered media stream is
   listed as sendrecv (or if there is no direction attribute at the
   media or session level, in which case the stream is sendrecv by
   default), the corresponding stream in the answer MAY be marked as
   sendonly, recvonly, sendrecv, or inactive. 
```

According to 3264 https://www.rfc-editor.org/rfc/rfc3264#section-6.1 , `sendonly` is qualified.

Reply:
Good catch. Let me include that back.